src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -1758,9 +1759,48 @@ def discriminant(self):
 
         return (MatrixSpace(QQ, 4, 4)(L)).determinant().sqrt()
 
+    def is_maximal(self):
+        r"""
+        Return ``True`` if the order is maximal in the ambient quaternion algebra, ``False`` otherwise
+        Only works in quaternion algebras over number fields
+
+        OUTPUT: Boolean
+
+        EXAMPLES::
+
+            sage: p = 11
+            sage: B = QuaternionAlgebra(QQ,-1,-p)
+            sage: i,j,k = B.gens()
+            sage: O0_basis = (1,i,(i+j)/2, (1+i*j)/2)
```

Comment:
fix spacing (after each comma there should be a space except in some specific cases)

Reply:
The main example is in one-element tuples, such as `(1,)`. I also tend to omit spaces when declaring multiple symbolic variables (`var('x,y')`), but I'm not sure if this is generally accepted.